node/src/test/scala/coop/rchain/node/mergeablity/MergeabilityRules.scala

```diff
@@ -296,4 +299,6 @@ trait BasicMergeabilityRules extends ComputeMerge {
         new TestFailedException(e, failedCodeStackDepth = 5).severedAtStackDepth
     }
   }
+
+  def shardId: String
```

Comment:
I tried to provide a maximum configurable environment. I [replaced](https://github.com/rchain/rchain/pull/3631/commits/02b423807c0d09c741f18f593cf583d6989c5b59) the method with just an empty string since this is not important.